internal/commands/root.go

```diff
@@ -305,3 +289,12 @@ func printByScanInfoFormat(cmd *cobra.Command, view interface{}) error {
 	f, _ := cmd.Flags().GetString(params.ScanInfoFormatFlag)
 	return printer.Print(cmd.OutOrStdout(), view, f)
 }
+
+func setUpFeatureFlags(featureFlagsWrapper wrappers.FeatureFlagsWrapper) {
+	err := wrappers.HandleFeatureFlags(featureFlagsWrapper)
```

Comment:
what did yo changed in here?

Reply:
I loaded all the feature flags at the start of the CLI because they are necessary for determining whether to include or exclude container descriptions in the CLI help and extensions.

Reply:
Still considering whether this is the optimal implementation.